docs/src/docs/welcome/faq.mdx

```diff
@@ -16,7 +16,7 @@ See [how to properly install `golangci-lint` in CI](/welcome/install/#ci-install
 
 1. Please, ensure you are using the latest binary release.
 2. Run it with `-v` option and check the output.
-3. If it doesn't help create a [GitHub issue](https://github.com/golangci/golangci-lint/issues/new) with the output from the error and #2 above.
+3. If it doesn't help create a [GitHub issue](https://github.com/golangci/golangci-lint/issues/new?assignees=&labels=bug&projects=&template=bug_report.yml) with the output from the error and #2 above.
```

Comment:
```suggestion
3. If it doesn't help create a [GitHub issue](https://github.com/golangci/golangci-lint/issues/new) with the output from the error and #2 above.
```

I prefer that the users have the option between Discussion and Bug.

Reply:
Added `/choose` that will open the following dialog:

<img width="1401" alt="image" src="https://github.com/golangci/golangci-lint/assets/3228886/6425fe12-950b-4cb5-bdf7-752b26d12a18">